frontend/src/components/taskSelection/taskActivity.js

```diff
@@ -59,21 +60,30 @@ const PostComment = ({ projectId, taskId, setCommentPayload }) => {
   );
 };
 
-export const TaskHistory = ({ projectId, taskId, commentPayload }) => {
+export const TaskHistory = ({ projectId, taskId, commentPayload, mapperLevel }) => {
   const token = useSelector((state) => state.auth.get('token'));
   const [history, setHistory] = useState([]);
+  const [showTaskComments, setShowTaskComments] = useState(false);
```

Comment:
It would be better to start both `showTaskComments` and `showTaskChanges` as true and make `showTaskChanges` as `false` if the user is begginer, so the user understands what is activated.